src/sentry/templates/sentry/base-react.html

```diff
@@ -18,3 +18,18 @@
     </div>
   </div>
 {% endblock %}
+
+{% block scripts_bottom %}
+<script>
+  function setBodyTheme() {
+    if (window.__initialData.user.options.theme) {
+      document.body.classList.add(`theme-${window.__initialData.user.options.theme}`)
+    }
+  }
+
+  try {
+    setBodyTheme();
+  } catch (_) {
+  }
```

Comment:
Nit, but inlining this inside the try catch would be fine too
```suggestion
  try {
    if (window.__initialData.user.options.theme) {
      document.body.classList.add(`theme-${window.__initialData.user.options.theme}`)
    }
  } catch (_) {
  }
```